src/main/kotlin/no/nav/klage/oppgave/api/mapper/OppgaveMapper.kt

```diff
@@ -193,46 +84,36 @@ class OppgaveMapper(
         )
     }
 
-    fun mapOppgaveKopiAPIModelToEsOppgave(oppgave: OppgaveKopiAPIModel): EsOppgave {
-        val fnr = oppgave.ident.folkeregisterident!!
-        val erEgenAnsatt = egenAnsattService.erEgenAnsatt(fnr)
-        val personInfo = pdlFacade.getPersonInfo(fnr)
+    fun mapToEs(klagebehandlingOgMottak: Pair<Klagebehandling, Mottak>): EsKlagebehandling {
```

Comment:
mapToEs var litt lite beskrivende. Map what to what?

Reply:
Har endret navn nå